docs/glossary.md

```diff
@@ -323,6 +377,10 @@ The amount of DCR one must time-lock in order to buy a ticket. The ticket price
 
 The process of buying part of a ticket. This is done by coordinating with other parties who will buy the other parts of a ticket. This can be done without giving up custody to your DCR. The minimum amount of DCR that can be put into a split ticket is 5 DCR. Ticket-splitting is currently coordinated through the 'ticket-splitting' channel on the [Decred slack](https://slack.decred.org/) channel. 
 
+#### Tor
+
+The Onion Router (Tor) is free open-source software which enables users to improve security and privacy when communicating over a network. It is commonly used to protect against Internet traffic analysis or to enable anonymous communication.
+
```

Comment:
Would a link to https://www.torproject.org/ be useful here?

Reply:
Sounds good to me. I've added a link.